hedera-node/hedera-app/src/main/java/com/hedera/node/app/statedumpers/accounts/AccountDumpUtils.java

```diff
@@ -84,7 +85,7 @@ public static BBMHederaAccount[] gatherAccounts(
     public static BBMHederaAccount fromMod(OnDiskValue<Account> account) {
         return new BBMHederaAccount(
                 account.getValue().accountId(),
-                account.getValue().alias(),
+                account.getValue().alias() != Bytes.EMPTY ? account.getValue().alias() : Bytes.EMPTY,
```

Comment:
Not needed here? if account.getValue().alias()==Bytes.EMPTY, it's still account.getValue().alias().

Reply:
Ah yes .. debug code. Thanks Iris. will remove now.

Reply:
Fixed. Thanks!